tutorials/poromechanics.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<p>Eq. 1 is supplemented by eq. 2, which is the thermo-poromechanical stress. Thus, the reference might be somewhat confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>